docs/devel/CONTRIBUTING.md

```diff
@@ -84,14 +84,14 @@ go: downloading github.com/giantswarm/crd-docs-generator v0.7.1
 Wrote /work/pkg/gadgettracermanager/containers-map/containersmap_bpfel.go
 ```
 
-### Building Local Gadget
+### Building IG
```

Comment:
nit: I'm not sure if it should be "Building `ig`" and the same for all the "With `ig`". What do other folks think?

Reply:
Removed every uppercase `IG`, except one in `cmd/kubectl-gadget/undeploy.go:167`. There it doesn't refer to `local-gadget`
```go
// Let's try to remove components of IG versions before v0.5.0,
// just in case somebody has a newer CLI but is trying to remove
// an old version of Inspektor Gadget from the cluster. Given
// that this is a best effort work, we don't track any error.

// kube-system/gadget daemon set
k8sClient.AppsV1().DaemonSets("kube-system").Delete(
context.TODO(), "gadget", metav1.DeleteOptions{},
)
```